common/app/views/fragments/atoms/mediaPosterOnly.scala.html

```diff
@@ -0,0 +1,17 @@
+@(media: model.content.MediaAtom)
+<figure
+itemprop="associatedMedia image"
+itemscope
+itemtype="http://schema.org/ImageObject"
+data-component="image" xmlns="http://www.w3.org/1999/html"
+class="element element-image">
+    <div class="u-responsive-ratio">
+            <img
```

Comment:
Thanks for letting me know about the `PosterImage` changes, I would rather merge this using the old CAPI model and pick up that work as a separate PR.